react-client/src/Components/App.js

```diff
@@ -71,30 +75,21 @@ function App() {
     });
 
     document.addEventListener("mouseover", async (e) => {
-      
-      const isNotDomInfoComponent = !domUtils.ancestorExistsByClassName(e.target, "dom-info-dialog-box");
-
-      if (isNotDomInfoComponent && e.target.nodeName !== "HTML") {
-
-        const domType = e.target.nodeName?.toLowerCase();
-
-        await setDomLeanDetails({ ...domLeanDetails, elId: e.target.id, domType, elClassNames: [...e.target.classList] }); //note: we used `await` implementation to wait for setState to finish setting the state before we append the React component to DOM. Not doing this would result in a bug and the DOM details we set in state won't be captured in the DOM.
-
-        e.target.classList.toggle("focused-dom");
-
-        e.target.appendChild(refDomHighlight.current.base);
-      }
+      // const isNotDomInfoComponent = !domUtils.ancestorExistsByClassName(e.target, "dom-info-dialog-box");
+      // if (isNotDomInfoComponent && e.target.nodeName !== "HTML") {
+      //   const domType = e.target.nodeName?.toLowerCase();
+      //   await setDomLeanDetails({ ...domLeanDetails, elId: e.target.id, domType, elClassNames: [...e.target.classList] }); //note: we used `await` implementation to wait for setState to finish setting the state before we append the React component to DOM. Not doing this would result in a bug and the DOM details we set in state won't be captured in the DOM.
+      //   e.target.classList.toggle("focused-dom");
+      //   e.target.appendChild(refDomHighlight.current.base);
+      // }
     });
 
     document.addEventListener("mouseout", (e) => {
-
-      const isNotDomInfoComponent = !domUtils.ancestorExistsByClassName(e.target, "dom-info-dialog-box");
-
-      if (isNotDomInfoComponent && e.target.nodeName !== "HTML") {
-        e.target.classList.toggle("focused-dom");
-
-        e.target.removeChild(refDomHighlight.current.base);
-      }
+      // const isNotDomInfoComponent = !domUtils.ancestorExistsByClassName(e.target, "dom-info-dialog-box");
```

Comment:
Why is this commented out??? Please restore.

Reply:
sorry nkalimot lng  already restored